docs/ossm/ossm2-migration/README.md

```diff
@@ -8,22 +8,117 @@ Before you begin to migrate your controlplane from OpenShift Service Mesh 2.6 to
+  - Default ingress/egress gateways are disabled. Follow [these instructions](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-gateway-migration) to migrate off of the default ingress/egress gateways.
 - Run the [migration-checker script](migration-checker.sh) to detect any issues with your environment.
 
 > [!WARNING]
 > You must upgrade your OpenShift Service Mesh 2 Operator to the latest release **before** you install the OpenShift Service Mesh 3 operator. If you upgrade your OpenShift Service Mesh 2 operator _after_ you install your OpenShift Service Mesh 3 operator, you will need to then uninstall and reinstall your OpenShift Service Mesh 3 operator to ensure the included CRDs are up to date.
 
+By the end of this checklist, your `ServiceMeshControlPlane` should look something like this:
+
+```yaml
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v2.6 # 1.
+  security: # 2.
+    manageNetworkPolicy: false
+  addons: # 3.
+    grafana:
+      enabled: false
+    kiali:
+      enabled: false
+    prometheus:
+      enabled: false
+  meshConfig: # 4.
+    extensionProviders:
+      - name: prometheus
+        prometheus: {}
+      - name: otel
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local
+  gateways: # 5.
+    enabled: false
+    openshiftRoute:
+      enabled: false
+  mode: MultiTenant
+  tracing: # 3.
+    type: None
+```
+
+1. Your `ServiceMeshControlPlane` has been updated to the latest version.
+2. Network policy management has been disabled.
+3. Addons and tracing have all been disabled.
+4. Your `ServiceMeshControlPlane` is configured to use external metrics and tracing providers.
+5. Managed gateways are disabled.
+
+You will also have a `Telemetry` resource in your root namespace e.g. `istio-system` that looks roughly like this:
+
+```yaml
+apiVersion: telemetry.istio.io/v1
+kind: Telemetry
+metadata:
+  name: mesh-default
+  namespace: istio-system
+spec:
+  metrics: # 1.
+    - providers:
+        - name: prometheus
+  tracing: # 2.
+    - providers:
+        - name: otel
+```
+
+1. Your metrics provider is specified. The `name` should match what is specified on your `ServiceMeshControlPlane` in the `spec.meshConfig.extensionProviders` field.
+2. Your tracing provider is specified. The `name` should match what is specified on your `ServiceMeshControlPlane` in the `spec.meshConfig.extensionProviders` field.
+
+And a `Kiali` resource that looks roughly like this:
+
+```yaml
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: istio-system
+spec:
+  version: default # 1.
+  external_services:
```

Comment:
Does it make sense to also mention istio.config_map_name ? Or maybe just point users to Kiali doc how to configure multiple istiods on kiali side?

Reply:
You don't need to specify `istio.config_map_name`. In fact, not specifying these will allow Kiali to do autodiscovery and work with both controlplanes. This was some work we did on the latest versions of Kiali. You should only need to specify the prom/tracing config since Kiali can't autodiscovery those as easily.